pkgs/servers/foundationdb/cmake.nix

```diff
@@ -31,23 +32,28 @@ let
           inherit rev sha256;
         };
 
-        buildInputs = [ libressl boost ];
-        nativeBuildInputs = [ cmake ninja python3 openjdk mono ]
+        buildInputs = [ ssl boost ]
+          ++ lib.optionals (lib.versionAtLeast version "7.1.0") [ msgpack toml11 ];
+
+        nativeBuildInputs = [ pkg-config cmake ninja python3 openjdk mono ]
```

Comment:
I initially added `pkg-config` to help resolve OpenSSL/LibreSSL, but we still need to explicitly tell CMake where to find them. I can remove `pkg-config` again if preferred.